doc/source/merging.rst

```diff
@@ -303,6 +303,28 @@ This is also a valid argument to ``DataFrame.append``:
           labels=['df1', 'df4'], vertical=True);
    plt.close('all');
 
+.. _merging.assign:
+
+Concatenating using ``assign``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can concatenate one Series to a DataFrame using the DataFrame method
```

Comment:
Can you add *when* you would do this.

Reply:
I'm not sure I understand what you want, so let me tell you why I'm sending this PR.

I learned about `assign` while trying to make a [`SettingWithCopyWarning`](http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy) warning go away. The proposed solution:

```python
df.loc[:, ('derived_column')] = value * df.loc[:,('X0')]
```

was still generating a warning in my case. So I've read the doc about `merging` and stumbled upon a `pd.concat` solution:

```python
df = pd.concat([df, value * df['X0']], axis=1)
df.columns.values[1] = "derived_column"
```

Comparing notes with a colleague she told me about `assign` and I found that solution more elegant:

```python
df = df.assign(derived_column = value * df['X0'])
```

So that's why I want to add it to `merging`. Now that I think about it, should I add a reference to `assign` in the [`indexing-view-versus-copy`](http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy) doc also? I didn't do it because I don't understand why my original solution using `.loc` still triggered the warning.

Back to your comment, I guess the *when* is when I want to add a column to a DataFrame but I already said that, no?

Reply:
After submitting my comment, I understood why the `.loc()` solution is causing a warning. I was already working on a view due to prior processing. To make sure I tested with:

```python
df = df.copy() # making sure it is a full df and not a view
df.loc[:, ('derived_column')] = value * df.loc[:,('X0')]
```

and it didn't produce a warning.